src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -163,5 +164,14 @@
             <argument type="service" id="http_kernel" />
             <argument>%twig.exception_listener.controller%</argument>
         </service>
+
+        <service id="twig.configurator.environment" class="Symfony\Bundle\TwigBundle\DependencyInjection\Configurator\EnvironmentConfigurator" public="false">
+            <argument />
+            <argument />
+            <argument />
+            <argument />
+            <argument />
+            <argument />
```

Comment:
Maybe to not confuse developers looking where those arguments are set add comment like:
```xml
<!-- Above arguments are set in TwigExtension -->